packages/dso-toolkit/src/components/header/header.models.ts

```diff
@@ -10,6 +10,8 @@ export interface Header {
   authStatus?: "none" | "loggedOut" | "loggedIn";
   loginUrl?: string;
   logoutUrl?: string;
+  showHelp?: boolean;
+  helpUrl?: string;
```

Comment:
Hier mag deze wel optioneel blijven: Bij het gebruiken van de template function is het voor ons niet verplicht om `true` of `false` te zetten. Als wij (toolkit maintainers) 'm niet zetten valt het component voor ons terug op zijn default.